.github/workflows/perf_sysbench.yml

```diff
@@ -45,5 +58,34 @@ jobs:
           name: perf_sysbench
           retention-days: 21
           path: |
-            *_result.txt
-            *_t_version.txt
+            ${{ github.workspace }}/bench-run/benchs/sysbench/*_result.txt
+            ${{ github.workspace }}/bench-run/benchs/sysbench/*_t_version.txt
+
+# TODO: set up publishing in same job
+  publish:
+    needs: perf_sysbench
+    # TODO: either fix post request availability from perf machines or
+    # decide which boxes could be used
+    runs-on: manual
+
+    steps:
+      - name: checkout tarantool
+        uses: actions/checkout@v2.3.4
+      - name: Download all workflow run artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: perf_sysbench
+          path: ${{ github.workspace }}/bench-run/publication/perf_sysbench
+      - name: install requirements
+        run: python3 -m pip install -r ${{ github.workspace }}/bench-run/requirements.txt
```

Comment:
I would add a call of the `setup-python` action before this step with python version 3.7, for example.

Reply:
added